addon/components/o-s-s/currency-input.hbs

```diff
@@ -1,16 +1,28 @@
 <div class="currency-input-container fx-1" ...attributes>
-  <div class="currency-input upf-input fx-row fx-1 fx-xalign-center">
-    <div class="currency-selector fx-row" role="button" {{on "click" this.toggleCurrencySelector}}>
-      <span class="margin-right-xxx-sm text-center symbol">{{this.selectedCurrencySymbol}}</span>
+  <div class="{{if this.onlyCurrencyInput "currency-input-onlycurrency" "currency-input"}} upf-input fx-row fx-1 fx-xalign-center ">
```

Comment:
```suggestion
  <div class="currency-input {{if this.onlyCurrencyInput "currency-input--currency-only"}} upf-input fx-row fx-1 fx-xalign-center ">
```
Keep `currency-input` class to avoid code duplication and update `currency-input--currency-only` to change only necessary css properties